.github/workflows/release.yml

```diff
@@ -54,34 +54,17 @@ jobs:
 
     - name: Install dependencies
       run: yarn install --frozen-lockfile
+
     - name: Build packages
       run: yarn build
+
     - name: Run unit tests
       run: yarn test
 
-    - name: List changed packages
-      run: yarn list-changed-packages
-
-    - name: Configure npm and git
-      run: |
-        yarn logout
-        echo "@superset-ui:registry=http://registry.npmjs.org/" > .npmrc
-        echo "registry=http://registry.npmjs.org/" >> .npmrc
-        echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
-        npm whoami
-        git config --local user.email "action@github.com"
-        git config --local user.name "GitHub Action"
-        git remote set-url origin "https://${GITHUB_TOKEN}@github.com/apache-superset/superset-ui.git" > /dev/null 2>&1
+    - name: Publish new versions found in package.json
       env:
         NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-    - name: Bump version and publish package(s)
       run: |
-        git update-index --assume-unchanged .npmrc
-        git tag -d `git tag | grep -E '^trigger-'`
-        yarn ci:release-from-tag
-      env:
-        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GH_TOKEN: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+        echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
+        npm whoami
+        yarn ci:release-from-tag  # lerna publish from-package
```

Comment:
Please do not change these yet. 
`yarn ci:release-from-tag` actually did not bump the package version, only publish to npm by comparing the versions in `package.json` and the latest version on `npm`.

I had called a command that actually bump the package version called before, but removed that temporarily due to conflict with protected `master` branch and I didn't have time to fix and test. These credentials that you have removed in the PR are necessary for the version bump, git push and create github release to succeed.

Reply:
Thought we'd be manually update package versions and only use `lerna publish from-package`. If we are going to automatically update versions and make commits in CI, should we publish with `--canary` instead?

Reply:
I sense it'd be quite troublesome if we let CI automatically make new commits. It'd basically mean every merge will create another commit in Git.

I like what [zeit/now is doing](https://github.com/zeit/now/blob/master/utils/publish.sh): publish stable versions from git tags, but canary versions from pushes to the master branch.

Reply:
Canary sounds good. We can discuss more. I was hesitating about publishing every version as well so I pulled it out.